pyconfig.h.in

```diff
@@ -557,9 +557,6 @@
 /* Define to 1 if you have the 'lchflags' function. */
 #undef HAVE_LCHFLAGS
 
-/* Define to 1 if you have the `lchmod' function. */
-#undef HAVE_LCHMOD
```

Comment:
Apparently `AC_CHECK_FUNCS` defines `HAVE_*` preprocessor macro when a function is found, but `AC_CHECK_FUNC` does not.
So `AC_CHECK_FUNC(lchmod)` should be changed to `AC_CHECK_FUNCS(lchmod)`.
Then above deletion from `pyconfig.h.in` should disappear.